config/categories.yaml

```diff
@@ -90,3 +90,12 @@ pillars:
       - name: 'Technology Products & Services'
         db_value: 'Technology Products & Services'
         slug: 'technology-products-services'
+
+  - name: 'General Common Goods and Services'
+    db_value: 'General Common Goods and Services'
+    slug: 'general-common-goods-and-services'
+
+    categories:
+      - name: 'Below Threshold'
+        db_value: 'Below Threshold'
+        slug: 'below-threshold'
```

Comment:
Where has this come from?

Reply:
https://crowncommercialservice.atlassian.net/browse/WEB-1448

Reply:
Should it show up in uat/prod if there's no frameworks associated with it? Let's get clarification from Nicola, as in the ticket she seems to suggest it's backend work that won't affect the users

Reply:
I will mention this on the group chat

Reply:
removed from this deployment